tests/models/whisper/test_modeling_whisper.py

```diff
@@ -1748,7 +1748,7 @@ def test_speculative_decoding_distil(self):
         processor = WhisperProcessor.from_pretrained(model_id)
 
         assistant_model_id = "distil-whisper/distil-large-v2"
-        assistant_model = WhisperForCausalLM.from_pretrained(
+        assistant_model = WhisperForConditionalGeneration.from_pretrained(
```

Comment:
The test should be kept as is here, we only want to load the decoder of distil-large-v2 here

Reply:
OK, I have reverted it.